sdks/wasm/debugger-locale-driver.html

```diff
@@ -0,0 +1,27 @@
+<!doctype html>
```

Comment:
Why is this new driver required? you can still use the `invoke_static_method ('[debugger-test] DebuggerTests.DateTimeTest:LocaleTest')` from `debugger-driver.html`.